python/python/bystro/prs/messages.py

```diff
@@ -2,11 +2,52 @@
 
 
 class PRSJobData(BaseMessage, frozen=True, forbid_unknown_fields=True, rename="camel"):
-    """Data for PRS jobs received from beanstalkd"""
+    """
+    The message to be sent through beanstalkd to initiate a PRS job
+
+    Attributes:
+    dosage_matrix_path: str
+        The path to the dosage matrix file
+    out_dir: str
+        The directory to save the output file
+    out_basename: str
+        The basename of the output file
+    assembly: str
+        The assembly to use for the PRS calculation
+    disease: str
+        The disease to use for the PRS calculation
+    pmid: str
+        The ID of the study (PubMed ID typically)
```

Comment:
We may want to include the GWAS catalog ID somewhere, and it might be a better choice if we stick to those overall since we could use that ID to interact with the GWAS catalog APIs using their study_accession_id
https://www.ebi.ac.uk/gwas/rest/docs/api